frontend/build-tc.sh

```diff
@@ -1,2 +1,7 @@
 #!/bin/bash
 echo ${BUILD_NUMBER:-DEV} > conf/build.txt
+
+# put the script needed on the machine into the package so it ends up on the instance via riffraff
```

Comment:
Maybe:
`put the init-instance.sh script into the riff-raff package so that it ends up in S3 ready to be fetched by the instance on boot`